tests/Session/test_profile.py

```diff
@@ -48,5 +56,13 @@ def testListFoodStorages(self):
         self.profile.addFoodStorage(FRIDGE, fridge_name)
         self.assertEqual([freezer_name, fridge_name], self.profile.listFoodStorages(FOOD_STORAGES)) 
 
+    def testAddFoods(self):
+        # Check self.profile.addFoods returns 0 = tests that food storage is not open yet 
+        self.profile.addFoodStorage(FREEZER, freezer_name)
+        self.assertIsNotNone(self.profile.getOpenFoodStorage())
+        
+        self.assertEqual(1, self.profile.addFoods([food1]))
+        # Test food gets added into inventory after list_foods command is implemented 
```

Comment:
I think this comment was meant to be moved up?

Reply:
Oh oops good catch lol